### ISSUE TITLE
Allow to stretch / squeeze transfer times based on the query

### ISSUE THREAD

**Issue by aubryio** [aubryio/minotor#46]
### Description

It should be possible to provider a multiplier (possibly <1) to the transfer times via the query to enable faster/slower transfers for accessibility reasons.
